### ISSUE TITLE
Linebrake (as per the official vsts reference) does not work

### ISSUE THREAD

**Issue by pekkaar** [cschleiden/vsts-markdown#11]
[This page explains the makrdown options supported by VSTS in general](https://www.visualstudio.com/en-us/docs/reference/markdown-guidance):

> In a Markdown file or widget, enter two spaces prior to the line break to begin a new paragraph, or enter two line breaks consecutively to begin a new paragraph.

This does not yet work in this widget. It would be a very nice addition. It is possible to simulate this by entering a new line, then a non-braking-space tag, then another new line:
```
[empty line]
&nbsp;
[empty line]
```

 but then going back to the view mode and attempting to edit the markdown text again throws a "convert" message, and when opting in, this 'hack' is removed.

**Comment by SeriousM** [comment]
yes, please have a fix for that!

**Comment by wjadevries** [comment]
Yes, all text with linebreaks shouldn't be continued on the same line. 
This is very annoying. 

I suggest one linebreak should be a `<br/>` and two should be a `<p>....</p>`

**Comment by iby-dev** [comment]
Shame, this bug is 2 yrs old and still not resolved yet.
